app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -7,6 +7,73 @@ def available_vms
     availability_zone.vms
   end
 
+  def self.params_for_create()
+    {
+      :fields => [
+        {
+          :component => 'switch',
+          :name      => 'shareable',
+          :id        => 'shareable',
+          :label     => _('Shareable'),
+          :onText    => _('Yes'),
+          :offText   => _('No')
+        },
+        {
+          :component    => 'select',
+          :name         => 'affinity_policy',
```

Comment:
This is not a valid field in the database, and I guess the API would not be able to deal with this either. Where is this information getting stored?

Reply:
We deferred affinity controls/inventory info in the current release. Perhaps we'll need to add a column to `cloud_volumes` next release?

I think the options will always be `['none', 'affinity', 'anti-affinity']`. Can we just hard-code these values into the schema here?

Reply:
If the column is not in the database, this form will just not work, so I'm removing this field until then...

Reply:
@skateman we're not create a cloud_volume database record, so not sure why it wouldn't work if the column doesn't exist?

Reply:
Wait, this form isn't creating a DB record?

Reply:
I assume this is the form that collects params that ends up being sent to `CloudVolume.create_volume_queue`

@jaywcarman ?

Reply:
Oh, in that case fields can be there, but then I need to hide them in edit mode...

Reply:
Correct - the form form data will go to `CloudVolume.create_volume_queue` where we make the IBM Cloud API call.

I like the idea of matching this form to the inventory picked up later by refresh, but yeah I was scratching my head on the hard requirement. I think everything makes sense now :)

Reply:
I will put back the removed fields then...thanks for the clarification.